helm/libero-reviewer/templates/deployment-continuum-auth.yaml

```diff
@@ -31,11 +31,19 @@ spec:
               value: "{{ .Release.Name }}-rabbitmq"
             - name: CONTINUUM_LOGIN_URL
               value: "{{ .Values.continuumLogin.url }}"
+            - name: AUTHENTICATED_REDIRECT_URL
+              value: "https://{{ include "libero-reviewer.fullname" . }}.elifesciences.org/auth"
+            - name: AUTHENTICATION_JWT_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "libero-reviewer.fullname" . }}
+                  key: liberoJwtSecret
```

Comment:
shouldn't they keys be `authenticationJwtSecret` and `continuumJwtSecret` to match the environment variables? No need to keep in mind another mapping